src/main/resources/data/modern_industrialization/recipes/vanilla_recipes/assembler/bell.json

```diff
@@ -0,0 +1,35 @@
+{
+    "type": "modern_industrialization:assembler",
+    "eu": 8,
+    "duration": 200,
+    "item_inputs": [
+        {
+            "item": "modern_industrialization:bronze_ring",
+            "amount": 1
+        },
+        {
+            "item": "modern_industrialization:bronze_plate",
+            "amount": 1
+        },
+        {
+            "item": "modern_industrialization:bronze_bolt",
```

Comment:
Bolts are not meant to be used in the assembler (they are replaced by soldering alloy).